src/index.html

```diff
@@ -5,10 +5,49 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto+Flex:opsz,wght@8..144,500&family=Roboto:wght@500&display=swap" rel="stylesheet">
     <title>Moyo header</title>
+
     <link rel="stylesheet" href="./style.css">
+    </style>
   </head>
-  <body>
-    <h1>Moyo header</h1>
+  <body id="home">
+
+    <header class="header"
+      >
+      <a class="logo" href="#"><img src="images/logo.png" alt="logo"></a>
+
+      <nav class="main-nav">
+        <ul class="nav__list">
+          <li class="nav_li">
+            <a href="#"  class="is-active main-nav-link">Apple</a>
+          </li>
+          <li class="nav_li">
+            <a href="#" class="main-nav-link">Samsung</a>
```

Comment:
Let's make the links more informative
```suggestion
          <li class="nav_li">
            <a href="#apple"  class="is-active main-nav-link">Apple</a>
          </li>
          <li class="nav_li">
            <a href="#" class="main-nav-link">Samsung</a>
```